<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STDF_viewer</class>
 <widget class="QMainWindow" name="STDF_viewer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>STDF_viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Records per Page</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>92</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>951</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="editPage">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>627</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string extracomment="0">0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNext">
    <property name="geometry">
     <rect>
      <x>1191</x>
      <y>622</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPrev">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>627</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>311</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Record Type</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listRecType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>231</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFilter">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>121</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Record Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineStart">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>306</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEnd">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>340</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>End Record Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEnd">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonClear">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>510</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Filter</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listFilter">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>141</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelStart_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter By Record Type</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textFilter">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>395</y>
       <width>151</width>
       <height>107</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>comma delimited</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStart_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>371</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>String Filters</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="txtBusy">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>751</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BUSY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>978</x>
      <y>634</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PAGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1083</x>
      <y>634</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPages">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>627</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string extracomment="0">0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
